test/remote/gateways/remote_monei_test.rb

```diff
@@ -0,0 +1,117 @@
+require 'test_helper'
+
+class RemoteMoneiTest < Test::Unit::TestCase
+  def setup
+    @gateway = MoneiGateway.new(
+        fixtures(:monei)
+    )
+
+    @amount = 100
+    @credit_card = credit_card('4000100011112224')
+    @declined_card = credit_card('5453010000059675')
+
+    @options = {
+      order_id: '1',
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+  end
+
+  def test_successful_purchase
+    response = @gateway.purchase(@amount, @credit_card, @options)
+
+    assert_success response
+    assert_equal 'Request successfully processed in \'Merchant in Connector Test Mode\'', response.message
+  end
+
+  def test_failed_purchase
+    response = @gateway.purchase(@amount, @declined_card, @options)
+    assert_failure response
+    assert_equal 'invalid cc number/brand combination', response.message
+  end
+
+  def test_successful_authorize_and_capture
+    auth = @gateway.authorize(@amount, @credit_card, @options)
+    assert_success auth
+
+    assert capture = @gateway.capture(@amount, auth.authorization)
```

Comment:
The newlines/lack of are all over the place in these tests, but I can clean that up in the merge